api/src/config/index.ts

```diff
@@ -59,7 +57,7 @@ const env = process.env.APP_ENV?.toString() || 'development';
 
 const dbURI: string =
   process.env.POSTGRES_URI ||
-  (process.env.VCAP_SERVICES && process.env.VCAP_SERVICES.match(PSQLREGEX)![1]) ||
+  (process.env.VCAP_SERVICES && JSON.parse(process.env.VCAP_SERVICES)["aws-rds"][0].credentials.uri) ||
```

Comment:
In demo, the only environment I tested on so far, the regex did not find the db uri.  `VCAP_SERVICES` has the same shape on demo as it does on stage and prod.  This should work on all 3 environments.